### PR TITLE
increased execution-time value

### PR DESCRIPTION
**Pull request by tehmas**
The default value of execution-time is 2700 seconds. Increasing this value to 7200 seconds will prevent time-out error for most of the people.

### CODE REVIEW

**Comment by poppyseedplehzr** [comment]
ok to test

File: tools/provision.ps1 [hunk 1]
```diff
@@ -288,6 +288,7 @@ function Install-ThirdParty {
   Try {
     foreach ($package in $packages) {
       $chocoForce = ""
+      $executionTime = 7200
```
Comment (mjpieters) [inline tools/provision.ps1]: Should this perhaps be named `executionTimeout` instead? As it is a maximum time limit, not a fixed amount of time.

Comment (poppyseedplehzr) [inline tools/provision.ps1]: Oo yeah, +1 this idea.

Comment (tehmas) [inline tools/provision.ps1]: You are right. Made the changes. Thanks!

**Pull request merged**